index.html

```diff
@@ -191,6 +275,70 @@
       </div>
     </div>
 
-  
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>
+    <script src="bower_components/jquery.hotkeys/jquery.hotkeys.js"></script>
```

Comment:
I looked for an offical CDN but didn't find one. Do you know of one that I don't?